### ISSUE TITLE
Why metrics goes better significantly at ~90 epoch?

### ISSUE THREAD

**Issue by dantetemplar** [ultralytics/ultralytics#16713]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Every time I train YOLO on VisDrone dataset, I see a sharp step near 90 epoch.

![image](https://github.com/user-attachments/assets/3959bd19-0233-497b-ace4-8ba832d4db10)

I'm not sure but LRs are seems to be linear: 

![image](https://github.com/user-attachments/assets/08869dcd-10ff-41c6-baf8-3dd166e01b73)

And train losses jump up too:

![image](https://github.com/user-attachments/assets/2ffbe7e8-f4c8-480a-bef3-32c0c34dd173)

But val losses vice versa - become lower:

![image](https://github.com/user-attachments/assets/47924658-9f5d-4889-99a4-007b57ebd0dd)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @dantetemplar, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com) for users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, and many common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it.

For custom training queries, please provide as much information as possible, including dataset image examples and training logs. You can also verify you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://ultralytics.com/discord) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com). Or dive into threads on our [Subreddit](https://reddit.com/r/ultralytics) to share knowledge with the community.

## Upgrade

To ensure your issue is not already resolved in the latest version, upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can run in any up-to-date verified environments with all dependencies, including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Please note this is an automated response. An Ultralytics engineer will assist you soon! 😊

**Comment by Y-T-G** [comment]
The last 10 epochs run without mosaic augmentations. So the training images become easier since mosaic is a strong augmentation.

This behavior can be controlled by setting the `close_mosaic` argument.

https://docs.ultralytics.com/modes/train/#train-settings

**Comment by dantetemplar** [comment]
> The last 10 epochs run without mosaic augmentations. So the training images become easier since mosaic is a strong augmentation.
> 
> This behavior can be controlled by setting the `close_mosaic` argument.
> 
> https://docs.ultralytics.com/modes/train/#train-settings

Thank you!

**Comment by glenn-jocher** [comment]
You're welcome! If you have any more questions, feel free to ask.
